src/PockyBot.NET/StartupExtensions.cs

```diff
@@ -27,7 +27,7 @@ public static class StartupExtensions
             return services;
         }
 
-        private static IServiceCollection ConfigureCommonServices(this IServiceCollection services) {
+        private static void ConfigureCommonServices(this IServiceCollection services) {
```

Comment:
Is there a reason you changed the signature of this extension? I'm just trying to be consistent with how these are usually written. I think they return themselves so that they can be chained or something.